client/components/accordion/style.scss

```diff
@@ -25,9 +25,18 @@ $accordion-background-expanded: $white;
 	color: $gray-dark;
 	background-color: $accordion-background-collapsed;
 	text-align: left;
+	transition: all 150ms ease-in;
```

Comment:
We can be more restrictive if we want yes. I tend to do "all" more often as there's no harm in animating other stuff if necessary.